services/core-web/src/components/Forms/permits/conditions/NestedListItem.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { Col, Row, Button } from "antd";
+import { UpOutlined, DownOutlined } from "@ant-design/icons";
+import { TRASHCAN, EDIT_OUTLINE_VIOLET } from "@/constants/assets";
+import NOWActionWrapper from "@/components/noticeOfWork/NOWActionWrapper";
+import * as Permission from "@/constants/permissions";
+import ListItemForm from "@/components/Forms/permits/conditions/ListItemForm";
+
+const propTypes = {
+  condition: PropTypes.objectOf(PropTypes.any),
+  new: PropTypes.bool,
+  handleSubmit: PropTypes.func,
+  handleCancel: PropTypes.func,
+  handleDelete: PropTypes.func,
+  reorderConditions: PropTypes.func,
+  setConditionEditingFlag: PropTypes.func,
+  initialValues: PropTypes.objectOf(PropTypes.any),
+  editingConditionFlag: PropTypes.bool.isRequired,
+  isViewOnly: PropTypes.bool,
+};
+
+const defaultProps = {
+  condition: undefined,
+  new: false,
+  handleSubmit: () => {},
+  handleCancel: () => {},
+  handleDelete: () => {},
+  reorderConditions: () => {},
+  setConditionEditingFlag: () => {},
+  initialValues: {},
+  isViewOnly: false,
+};
+
+const NestedListItem = (props) => {
+  // eslint-disable-next-line no-unused-vars
+  const [isEditing, setIsEditing] = useState(props.new);
+  return (
+    <>
+      <Row gutter={[8, 16]} className={isEditing || props.isViewOnly ? " " : "hover-row"}>
```

Comment:
Can we set `className` to `""` rather than an empty space?

Reply:
removed